rcongui/Dockerfile

```diff
@@ -2,26 +2,30 @@ FROM node:13-stretch
 
 WORKDIR /code
 
-RUN apt-get update -y && apt-get install -y nginx apache2-utils
+RUN apt-get update -y && apt-get install -y yarn nginx apache2-utils
 COPY nginx.conf /etc/nginx/sites-available/default
 
 RUN mkdir /pw
 
 COPY package.json .
-COPY package-lock.json .
+COPY yarn.lock .
 
-RUN npm install
+RUN yarn install
 
 ENV REACT_APP_API_URL /api/
 
 COPY src/ src/
 COPY public/ public/
-
-RUN npm run build
-
 COPY entrypoint.sh .
+
 RUN chmod +x /code/entrypoint.sh
 
+RUN yarn run build
+
+RUN rm -rf src/
+RUN rm -rf public/
+RUN rm -rf node_modules/
```

Comment:
By doing that you also screw with the caching right? Did you test?

Reply:
Yes I tested. No cache screwed at all :)

Reply:
Ok, so the installed packages will still be in the previous layers on the build machine but not in the final image? Good one then